app-devel/gcc-13/autobuild/beyond

```diff
@@ -0,0 +1,42 @@
+# WARNING!
+# Installing GCC runtime libraries to libdir/gcc/$HOST_TRIPLE/$PKGVER.
+abinfo "Determining the path to the installed shared libraries ..."
+SHLIBDIR=$(find $PKGDIR -maxdepth 3 -name libgcc_s.so.1 -printf '%h')
+
+
+abinfo "Setting executable bit for libgcc_s.so.1 ..."
+chmod -v +x "$SHLIBDIR"/libgcc_s.so.1
+
+abinfo "Removing libtool archives ..."
+find "$SHLIBDIR" -maxdepth 1 -name '*.la' -exec rm -v {} \;
+
+GCC_LIBDIR="$LIBDIR"/gcc/"${ARCH_TARGET[$ABHOST]}"/"$PKGVER"
+# Make sure the directory is what we wanted to install the libraries.
+if ! [ -e "$PKGDIR"/"$GCC_LIBDIR"/crtbegin.o ] ; then
+	abdie "Unable to find the initialization routine object files!"
+fi
+
+abinfo "Installing runtime files to GCC version-specific libdir ..."
+find "$SHLIBDIR" -maxdepth 1 -name '*.so'   -exec cp -av {} "$PKGDIR"/"$GCC_LIBDIR"/ \;
+find "$SHLIBDIR" -maxdepth 1 -name '*.so.*' -exec cp -av {} "$PKGDIR"/"$GCC_LIBDIR"/ \;
+find "$SHLIBDIR" -maxdepth 1 -name '*.a'    -exec cp -av {} "$PKGDIR"/"$GCC_LIBDIR"/ \;
+
+abinfo "Removing libraries from libdir ..."
+find "$SHLIBDIR" -maxdepth 1 -type f -exec rm -v {} \;
+find "$SHLIBDIR" -maxdepth 1 -type l -exec unlink {} \;
+
+if [ -d "$(dirname $SHLIBDIR)/lib64" ] ; then
```

Comment:
This shouldn't be needed since we already handled libdir in prepare?

Reply:
The runtime libraries are installed to `PREFIX/lib64`. `prepare` only sets the search directories for Go runtimes.